internal/config/logs.go

```diff
@@ -7,13 +7,15 @@ import (
 type Logs struct {
 	*API
 
-	Job    string
-	Stream string
+	Job       string
+	Stream    string
+	TailLines uint
 }
 
 func (c *Logs) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVarP(&c.Job, "job", "j", "", "job name where to get the attempt Logs")
+	cmd.Flags().StringVarP(&c.Job, "job", "j", "", "job name whose log lines will be displayed")
 	cmd.MarkFlagRequired("job")
 
-	cmd.Flags().StringVarP(&c.Stream, "stream", "s", "stdout", "channel where get the logs stdout or stderr")
+	cmd.Flags().StringVarP(&c.Stream, "stream", "s", "stdout", "standard stream from where to display log lines (stdout or stderr)")
```

Comment:
would suggest to remove the word "standard" from "standard stream"

Reply:
It's done here: https://github.com/signadot/cli/pull/122/commits/9cb8a136216ca571c3906643f2272c647e7aada4.